### ISSUE TITLE
Error Code :  "not enough values to unpack"

### ISSUE THREAD

**Issue by matteozoia4** [deeptools/pyGenomeTracks#369]
Dear pyGenomeTracks providers,

I thought the error below was related to '_' inserted in the file names :

**pygenometracks.utilities.InputError: Error reading line: 1
Error message: not enough values to unpack (expected 3, got 1)**

After changing those files names I still obtain the identical error. Do you see what I am missing?

**Here I attach my .ini file if this can help!**
[tracksTrial1.txt](https://github.com/deeptools/pyGenomeTracks/files/7862866/tracksTrial1.txt)

Best regards and many thanks for the help!

M.Z

**Comment by lldelisle** [comment]
Hi @matteozoia4,
I think it is because your TadDomains.bed is separated by space and not tab. pyGenomeTracks only support bed separated by tabulations.
For the future bug report, would you mind to provide which version of pygenometracks you are using and the full command line and log that you get?
Thank you very much

**Comment by lldelisle** [comment]
Indeed it is not a space/tab issue.
I cannot reproduce the error. Would you mind to either try with smaller ini files with one track at a time or to send me by email your input files?
Thanks

**Comment by lldelisle** [comment]
Hi,
the 2 files `vlinesforTadDomains.bed`, `TadDomains.bed` and `4CarcsInteractionsOUTSIDEmap.arcs` were rich text format:
```
{\rtf1\ansi\ansicpg1252\cocoartf2636
\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
{\*\expandedcolortbl;;}
\paperw11900\paperh16840\margl1440\margr1440\vieww18920\viewh8400\viewkind0
\deftab720
\pard\pardeftab720\partightenfactor0

\f0\fs24 \cf0 \expnd0\expndtw0\kerning0
chr8	59000000	60000000	active	2	.	59000000	60000000	255,255,153}
```
If I use `plain text` format, I can solve this error.
I should add an error message specific to the header, this may help future users.

**Comment by lldelisle** [comment]
Solved in version 3.7
